keyboards/kprepublic/bm40hsrgb/keymaps/dan/keymap.c

```diff
@@ -18,56 +18,69 @@
 
 enum layers {
     _ALPHA,
+    _WMAN,
     _LEFTFN,
     _NUMPAD,
     _RIGHTFN,
     _MOBA,
     _FPS1,
-    _FPS2,
     _STRAT,
     _ARROWS,
     _CLEAN,
-    _RGB
+    _MOU
 };
 
 #define ALPHA TO(_ALPHA)
+#define WMAN TG(_WMAN)
 #define LEFTFN MO(_LEFTFN)
 #define NUMPAD MO(_NUMPAD)
 #define RIGHTFN MO(_RIGHTFN)
 #define MOBA TG(_MOBA)
 #define FPS1 TG(_FPS1)
-#define FPS2 TG(_FPS2)
 #define STRAT TG(_STRAT)
 #define ARROWS TG(_ARROWS)
 #define CLEAN TG(_CLEAN)
-#define RGB MO(_RGB)
+#define MOU MO(_MOU)
 
-// Tap dance declarations
-enum {
-    TD_PAST_PSLS,
-    TD_PASTE_COPY
-};
+// // Tap Dance
+// enum {
+//     TD_PAST_PSLS,
+//     TD_PMNS_PPLS,
+// };
 
-tap_dance_action_t tap_dance_actions[] = {
-    [TD_PAST_PSLS]  = ACTION_TAP_DANCE_DOUBLE(KC_PAST,    KC_PSLS),
-    [TD_PASTE_COPY] = ACTION_TAP_DANCE_DOUBLE(LCTL(KC_V), LCTL(KC_C))
-};
+// tap_dance_action_t tap_dance_actions[] = {
```

Comment:
We also do not want commented out code in the repo, which there is a fair amount introduced in this file.